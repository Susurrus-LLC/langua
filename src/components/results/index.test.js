import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Results from '.'

describe('Results', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Results>Test</Results>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders accurately with and without children', () => {
    let tree = renderer.create(<Results />).toJSON()
    expect(tree).toMatchSnapshot()

    tree = renderer.create(<Results>Test</Results>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
