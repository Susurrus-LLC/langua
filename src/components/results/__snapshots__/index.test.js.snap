// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results renders accurately with and without children 1`] = `
<div
  className="results"
/>
`;

exports[`Results renders accurately with and without children 2`] = `
<div
  className="results"
>
  Test
</div>
`;
